www/src/components/shell/onboarding/OnboardingTips.js

```diff
@@ -0,0 +1,72 @@
+import { A, Code, Div, P } from 'honorable'
+import { TipCarousel } from 'pluralsh-design-system'
+
+function TipCode(props) {
+  return (
+    <Code
+      backgroundColor="fill-two"
+      font-style="normal"
+      paddingTop="0.1em"
+      paddingBottom="0.2em"    
+      paddingHorizontal="0.5em"
+      marginHorizontal="0.1em"
+      borderRadius="normal"
+      border="1px solid border-fill-two"
+      {...props}
+    />
+  )
+}
+
+function OnboardingTips({ autoAdvanceTime = 10000, ...props }) {
+  return (
+    <Div {...props}>
+      <TipCarousel
+        autoAdvanceTime={autoAdvanceTime}
+      >
+        {[
+          <>
+            You can use service accounts to have an entire team manage a set installations for one of your plural clusters. Learn more{' '}
+            <A
+              inline
+              target="_blank"
+              href="https://docs.plural.sh/identity-and-access-management/identity-and-installations/service-accounts"
+            >
+              here
+            </A>.
+          </>,
+          <P margin={0}>
+            Plural can be an identity provider for most web facing applications using OpenId Connect. Learn more{' '}
+            <A
+              inline
+              target="_blank"
+              href="https://docs.plural.sh/basic-setup-and-deployment/openid-connect"
+            >
```

Comment:
It's our documentation but it would be nice to have `rel="noopener noreferrer"` for all external links. It will also reduce number of warnings in the console.